workerbase/lib/build_crosscompiler.sh

```diff
@@ -168,6 +182,20 @@ install_kernel_headers()
     sudo -E rm -rf linux-${linux_version}
 }
 
+## Function to download and install FreeBSD headers
+install_freebsd_headers() {
+    freebsd_url="https://download.freebsd.org/ftp/releases/amd64/11.1-RELEASE/base.txz"
```

Comment:
FYI regarding using 11.1: As far as I know, building Julia on FreeBSD 10.x hasn't been attempted since the early days, when Mike Nolta was managing FreeBSD support. We currently build and run on 11.0 (and indeed the FreeBSD CI workers are currently running 11.0), but that has [reached EOL](https://lists.freebsd.org/pipermail/freebsd-announce/2017-October/001808.html), so I've listed the minimum version here as 11.1, which is the currently supported release.